Squeezer/src/main/res/values/strings.xml

```diff
@@ -23,6 +23,7 @@
     <string name="disabled">Disabled</string>
     <string name="pause">Pause</string>
     <string name="mute">Mute</string>
+    <string name="delete">Delete</string>
```

Comment:
We need other languages
Add ALARM_DELETE to slimstrings in build.gradle
or check Android sources

Reply:
I don't understand what you mean here. After all, everything in `strings.xml` has translated counterparts?

Reply:
The word “Delete” (in `<string name="delete">Delete</string>`) is not translated to any other language than English. My suggestion is that these translations are readily present in `serverstrings/slimserver/strings.txt`, which I copied from the server sources.

Reply:
OK, I see. I thought those have a special meaning. Adjusted as suggested.

Reply:
Actually, I meant to include these:
```
ALARM_DELETE
	CS	Odstranit budík
	DA	Slet vækkeur
	DE	Wecker entfernen
	EN	Remove Alarm
	ES	Quitar alarma
	FI	Poista herätys
	FR	Supprimer le réveil
	IT	Rimuovi sveglia
	NL	Weksignaal verwijderen
	NO	Fjern vekking
	PL	Usuń alarm
	RU	Удалить будильник
	SV	Ta bort alarm
```
But this is also ok.